README.md

```diff
@@ -23,3 +23,11 @@ To make testing close to a real-world scenario, an additional VM reboot step in
 
 > Please don't forget to include respective tests for every new critical feature of the system!<br>
 > See https://www.inspec.io/docs/reference/dsl_inspec/ and existing `/tests` examples which makes easy to add more tests.
+
+### `st2-integration-tests`
```

Comment:
@warrenvw I couldn't find a better naming, except of `st2-integration-tests`.
Let me know if you have better ideas/variations for such executable naming.

Reply:
The name `st2-integration-tests` is descriptive. For that reason, I like it. I can't think of anything better at the moment.